TestCases/rans_uq/naca0012/turb_NACA0012_uq.cfg

```diff
@@ -112,10 +112,10 @@ LINEAR_SOLVER= FGMRES
 LINEAR_SOLVER_PREC= LU_SGS
 %
 % Minimum error of the linear solver for implicit formulations
-LINEAR_SOLVER_ERROR= 1E-10
+LINEAR_SOLVER_ERROR= 0.05
 %
 % Max number of iterations of the linear solver for the implicit formulation
-LINEAR_SOLVER_ITER= 2
+LINEAR_SOLVER_ITER= 10
```

Comment:
@jayantmukho I changed your testcases as the very low tolerance + few iterations were keeping the CFL at the minimum level with the revised linear system logic.

Reply:
Yeah, I realized that with these new checks, we will have to revise the "best practices" suggestions. It used to be to keep `LINEAR_SOLVER_ERROR = 1E-10` and `LINEAR_SOLVER_ITER = 10`, essentially forcing 10 iterations every time. The CFL would increase as long as the residual of the linear solver was < 0.5.

With more sophisticated logic, we can probably suggest `LINEAR_SOLVER_ERROR=1E-6` and let the adaptive CFL do the work. 

Since you've implemented a hard tolerance of 0.001 in the code, wouldn't this value of 0.05 get overridden?

Reply:
The 0.001 is the lowest I allowed `max(0.001, config->GetLinear_Solver_Error());`
I don't think those are best practices, not in my experience anyway (which was what I based myself on to write this https://su2code.github.io/docs_v7/Linear-Solvers-and-Preconditioners/)

Reply:
I am embarrassed to say I hadn't read that yet. I had been going off the suggestions in the #790 

To the point you make in that guide, perhaps 0.001 is too high of a limit. Especially if running on adapted meshes where mesh quality is pretty poor. I am testing this before I make any claims.

Reply:
Cool let me know how that goes, I have no experience there.
10 iterations is a perfectly fine number for FGMRES if it allows decent CFL. The linear residual you can usually get with that on RANS grids won't be 1e-6.